WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java

```diff
@@ -159,6 +160,23 @@ private void loadImageIfNecessary(final boolean isInLayoutPass,
             return;
         }
 
+        int width = getWidth();
```

Comment:
Just a note for future reference: we actually used to have this code here, but it was removed in [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/4734) because it seemed to prevent the wrong image from appearing when used in lists. I think that was probably a red herring, though, given we still get reports of wrong gravatars, etc.